### PR TITLE
Added code to generate a classification report 

### PR DESCRIPTION
**Pull request by simonada**
Hi @jacobgil,

This pull request aims to allow users to generate a classification report similar to those created by [scikit-learn](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html), but with additional functionality to include confidence intervals for each metric. Leveraging the existing implementations within your library, it computes precision, recall, and F1-score for each class, both individually and as micro and macro averages.

Please review for any potential errors, and don't hesitate to suggest any improvements.

Much appreciated, and thank you for your outstanding package!

### CODE REVIEW

**Comment by jacobgil** [comment]
You ROCK, this is a great contribution.

Does it make sense to add a section with this to the Readme? Maybe with an example dataframe output?

**Comment by simonada** [comment]
Hi @jacobgil, thanks for the feedback :) 

I made some small fixes to the code and added a section in the Readme file.
